mobile-menu.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-multi-str */
+/* eslint-disable no-plusplus */
+const hdr = document.querySelector('.pg-hdr'); // the header element
+const btn = document.querySelector('.btn'); // the opening button
+const logo = document.querySelector('.pg-hdr h3'); // The logo name
+const navBar = document.querySelector('.dsk-btn'); // The desktop navigation bar
+const navLinks = document.querySelectorAll('.nav-link'); // The navigation links inside the nav bar
+const rightArrow = document.querySelectorAll('.r-arrow'); // The right arrow after the nav links
+const navIcon = document.querySelector('.nav-icon'); // The mail icon
+
+/** ******* The closig button *********** */
+const cross = document.createElement('button'); // The cross button to close mobile menu
+cross.setAttribute('style', '\
+width: max-content;\
+background: none;\
+font-weight: 500;\
+font-size: 38px;\
+border: none;\
+align-self: end;\
+cursor: pointer;');
+cross.innerHTML = '&times;';
+
+/** ********* The opening function ************** */
+function open() {
+  hdr.setAttribute('style', '\
+  height: 100vh;\
+  width: 100%;\
+  margin-top: 0;\
+  position: fixed;\
+  padding: 40px;\
+  background-color: #191765;\
+  flex-direction: column-reverse;\
+  justify-content: start;\
+  align-items: initial;');
+
+  navBar.setAttribute('style', '\
+  display: flex;\
+  flex-direction: column;\
+  width: 100%;\
+  max-width: 100%;\
+  height: 184px;\
+  font-size: 24px;\
+  padding: 16px 15px;\
+  margin-top: 18px;');
+
+  navIcon.setAttribute('style', 'display: none;');
+
+  logo.setAttribute('style', 'display: none;');
+
+  btn.setAttribute('style', 'display: none');
+
+  hdr.appendChild(cross); // add cross to the header
+
+  for (let i = 0; i < rightArrow.length; i++) { // Add the right arrow to every nav link
+    rightArrow[i].setAttribute('style', 'display: initial;');
+  }
+}
+
+btn.addEventListener('click', open); // calling the opening function
+
+/** *********** The closing function *************** */
+function close() { // Reseting the header to the original
+  hdr.removeAttribute('style');
+  btn.removeAttribute('style');
+  hdr.removeChild(cross);
+  logo.removeAttribute('style');
+  navBar.removeAttribute('style');
+  navIcon.removeAttribute('style');
```

Comment:
- [ ] Also assign this to a class style, then toggle it on the element.

Reply:
This issue is well resolved! ✔️